### DISCUSSION TITLE
Fetch private org projects

### DISCUSSION THREAD

**Discussion by ghosty2004** [octokit/octokit.js#2691] [Q&A] [answered]
Hi 👋 
Does anyone know how can I fetch a private organization projects?

**Comment by gr2m** [comment, marked as answer]
You have to make sure you have the right permission on your token (`read:org` scope, or `organization_projects:read` permission). 

Then you use GraphQL, e.g. for org projects

```gql
query orgProjects($org:String!) {
  organization(login:$org) {
    projectsV2(first:100) {
      nodes {
        title
        url
        public
      }
    }
  }
}
```

There is no way to filter out projects directly, so you would need to do that after loading all of them
